### ISSUE TITLE
about the modified and original codes...

### ISSUE THREAD

**Issue by chLFF** [gangsterless/pytorch-softmax-splatting#3]
# Modified parts and original parts...

You mentioned that many codes are from Niklaus. I wonder which parts of these codes are modified by you and which parts are original. Or can you send me the original codes from Niklaus that you got to this email 965697820@qq.com?
Thank you very much!

你在md中提到了很多代码来自于原作者Niklaus，请问能够说明项目里面哪一部分是修改过哪一部分是源码吗？或者你能够把你得到的Niklaus的源码发邮件给我吗？
十分感谢！

**Comment by gangsterless** [comment]
主要是官方的softmax splatting以及 [pytorch-pwc](https://github.com/sniklaus/pytorch-pwc)
